label_studio_converter/converter.py

```diff
@@ -397,13 +401,6 @@ def convert_to_coco(self, input_data, output_dir, output_image_dir=None, is_dir=
                 else:
                     raise ValueError("Unknown label type")
 
-                if category_name not in category_name_to_id:
-                    category_id = len(categories)
-                    category_name_to_id[category_name] = category_id
-                    categories.append({
-                        'id': category_id,
-                        'name': category_name
-                    })
                 category_id = category_name_to_id[category_name]
```

Comment:
also can you please soften this operation by using `category_name_to_id.get(category_name)`